GetStream/stream-chat-android
# Deprecate `DeletedMessageListItemPredicate`

### 🎯 Goal

Include`DeletedMessageListItemPredicate` in the same deprecation timeline as `MessageListView.setDeletedMessageListItemPredicate()`.

### 🛠 Implementation details

Set a deprecation annotation with `Level.Warning` and nclude `DeletedMessageListItemPredicate` in the deprecations markdown file.

### 🧪 Testing

1. Run the apps and make sure they run fine
2. Check the deprecation label at call site and make sure it is readable and understandable

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![out](https://user-images.githubusercontent.com/37080097/165495266-4153707e-1a2f-4140-8ba1-ad7517f6e588.gif)
